lib/ably/realtime/channel.rb

```diff
@@ -179,15 +184,26 @@ def presence
         @presence
       end
 
-      # Return the message history of the channel.
+      # Return the message history of the channel
+      #
+      # Once attached to a channel, you can retrieve messages published on the channel before the
```

Comment:
I think the documentation should mention the following points:
1- `history` is only a wrapper for the rest channel history method. It therefore takes the same `options` as the REST `history` call. In addition since it is evented, you can pass a block as a callback once the REST call is complete.
2- You can use the `:until_attach` option if you need to display history with a guarantee that no message will be missed.
3- To use `until_attach` you need to make sure that the channel is attached before calling `history`.
4- If the channel is not attached, this option will throw an exception.